integration-tests/tests.py

```diff
@@ -21,18 +21,43 @@ def run_tests(behaviour, args):
             if exists[0] != exists[1]:
                 sys.exit(f"{exists[0]} of {exists[1]} keys, {behaviour} test failed, see output")
             else:
-                print(f"{exists[0]} of {exists[1]} keys")   
+                print(f"{exists[0]} of {exists[1]} keys")
         else:
             sys.exit('Unknown behaviour.')     
     except subprocess.CalledProcessError as e:
         sys.exit(str(e.stderr))
     except Exception as e:
         sys.exit(str(e))
 
+def run_doubled_exist_test(args):
+    try:
+        print(f'Running bobp -b exist {args.rstrip()}')
+        p = subprocess.check_output(shlex.split(f'./bobp -b exist {args.rstrip()}')).decode('ascii')
+        print(str(p))
+        found_exist = re.search(r'\b[0-9]{1,}\sof\s[0-9]{1,}\b', str(p))
+        if not found_exist:
+            sys.exit(f"No exist output captured, check output")
+        exists = found_exist.group(0).split(' of ')
+        if int(exists[0]) * 2 + 1 != int(exists[1]):
```

Comment:
Bad check. Pass `expected_exists_keys` and compare with its value: `if int(exists[0]) != expected_exists_keys:`